SavingsManagementSystem.Common/DTOs/ChangePasswordRequest.cs

```diff
@@ -2,6 +2,8 @@
 {
 	public class ChangePasswordRequest
 	{
+		public string UserId { get; set; }
```

Comment:
Get the userId from the current logged-in user. You have to be logged in to change your password